downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -404,7 +404,7 @@ abstract class VersionControlSystem {
      * Check whether the VCS tool is at least of the specified [expectedVersion], e.g. to check for features.
      */
     fun isAtLeastVersion(expectedVersion: String): Boolean {
-        val actualVersion = Semver(getVersion(), Semver.SemverType.LOOSE)
-        return actualVersion.isGreaterThanOrEqualTo(expectedVersion)
+        val actualVersion = Semver.coerce(getVersion())
+        return actualVersion.isGreaterThanOrEqualTo(Semver.coerce(expectedVersion))
```

Comment:
Note for myself: `SemverType.LOOSE` was implicitly used before as well for `expectedVersion`.